matchzoo/models/match_lstm.py

```diff
@@ -0,0 +1,83 @@
+"""Match LSTM model."""
+import keras
+import keras.backend as K
+
+from matchzoo import engine
+
+
+class MatchLSTM(engine.BaseModel):
+    """
+    Match LSTM model.
+
+    Examples:
+        >>> model = MatchLSTM()
+        >>> model.guess_and_fill_missing_params(verbose=0)
+        >>> model.build()
+        >>> model.params['fc_hidden_size'] = 200
+        >>> model.params['rnn_hidden_size'] = 256
+
+    """
+
+    @classmethod
+    def get_default_params(cls):
+        """Get default parameters."""
+        params = super().get_default_params(with_embedding=True)
+        params.add(engine.Param(
+            'rnn_hidden_size', 256,
+            hyper_space=engine.hyper_spaces.quniform(low=128, high=384, q=128)
```

Comment:
`q` is step size. Should be 1 here.

Reply:
I choose a smaller step value

Reply:
I think it is not necessary to search every possible value for the hidden size

Reply:
@lixinsu It's not a grid search. `q` merely affects the granularity of sampled values.